### PR TITLE
small code cleanup

### PR DESCRIPTION
**Pull request by sergeylanzman**

### CODE REVIEW

File: pkg/cloudinstances/cloud_instance_group.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func GetNodeMap(nodes []v1.Node) map[string]*v1.Node {
 	nodeMap := make(map[string]*v1.Node)
 	for i := range nodes {
 		node := &nodes[i]
-		nodeMap[node.Spec.ExternalID] = node
+		nodeMap[node.Spec.ProviderID] = node
```
Comment (justinsb) [inline pkg/cloudinstances/cloud_instance_group.go]: This one isn't obviously safe to me - do you want to pull this one into a separate PR maybe?

File: protokube/pkg/gossip/mesh/mesh.pb.go [hunk 2]
```diff
@@ -235,7 +235,7 @@ func sovMesh(x uint64) (n int) {
 	return n
 }
 func sozMesh(x uint64) (n int) {
-	return sovMesh(uint64((x << 1) ^ uint64((int64(x) >> 63))))
+	return sovMesh(uint64((x << 1) ^ uint64(int64(x)>>63)))
```
Comment (justinsb) [inline protokube/pkg/gossip/mesh/mesh.pb.go]: Sadly this is generated code, so we shouldn't change it

File: upup/pkg/kutil/convert_kubeup_cluster.go [hunk 3]
```diff
@@ -480,7 +480,7 @@ func (x *ConvertKubeupCluster) Upgrade() error {
 		return fmt.Errorf("error writing completed cluster spec: %v", err)
 	}
 
-	oldCACertPool, err := oldKeyStore.CertificatePool(fi.CertificateId_CA, true)
+	oldCACertPool, err := oldKeyStore.FindCertificatePool(fi.CertificateId_CA)
```
Comment (justinsb) [inline upup/pkg/kutil/convert_kubeup_cluster.go]: This one isn't obviously safe (because FindCertificatePool won't create the certificate if it wasn't found.)  Split it out into a new PR?

**Comment by justinsb** [comment]
Some nice changes @sergeylanzman - thank you!

There are 2 changes that I'm not so sure about - maybe split them into separate PRs so we don't hold up the others?

And the change to the generated .pb.go code file will just conflicts, so can we leave that one as is as well please?

/ok-to-test

**Comment by justinsb** [comment]
Would like to get this in to 1.11, but it's too late for 1.10

**Comment by justinsb** [comment]
@sergeylanzman have you signed the CLA?  Maybe you changed the email on your commits at some stage?

**Comment by sergeylanzman** [comment]
@justinsb  
Hey
I created PR with other email. 
Today I am Googler and I don't need sign the CLA
